lib/chat_api_web/controllers/billing_controller.ex

```diff
@@ -17,6 +17,7 @@ defmodule ChatApiWeb.BillingController do
     with %{account_id: account_id} <- conn.assigns.current_user,
          account <- Accounts.get_account!(account_id),
          {:ok, _account} <- Billing.create_subscription_plan(account, plan) do
+      notify_slack(conn, plan)
```

Comment:
i suppose a slightly more "elixir"-y way of writing this could be:
```elixir
conn |> notify_slack(plan) |> json(%{data: ...})
```
what do you think?

Reply:
Thanks for the suggestion. This was definitely what I was looking for, but I just couldn't figure out! 👍